src/bundle/Resources/views/admin/section/list.html.twig

```diff
@@ -35,19 +35,19 @@
                             <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#create"></use>
                         </svg>
                     </a>
+                    {% set modal_data_target = 'delete-sections-modal' %}
+                    <button id="delete-sections" type="button" class="btn btn-danger" disabled data-toggle="modal"
```

Comment:
In order to improve code readability, please move each attribute onto a separate line.

Reply:
@sunpietro done